### ISSUE TITLE
Device 1 is not a termcap terminal device

### ISSUE THREAD

**Issue by tad-lispy** [clojure-emacs/parseclj#28]
Hi! Thanks for your work on `parseclj` and `parseedn`. I&rsquo;m still a novice at Emacs and hope you can help me. Also, I wasn&rsquo;t sure in which repository to file this issue, but I suspect this one is where the problem originates.

I&rsquo;ve installed the `parseedn` package and tried to use it on a simple value, like this:

``` emacs-lisp
(parseclj-read-str "42")
```

It gives me no result and the following message:

> Device 1 is not a termcap terminal device

Same for `parseclj-read-str`, `parseedn-print-str` and `parseclj-print-str`.

I&rsquo;m using Emacs Doom on NixOS with Emacs daemon (but I tried it without daemon running and the effect was the same).

``` emacs-lisp
(emacs-version)
```

> GNU Emacs 27.2 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.27, cairo version 1.16.0)

I&rsquo;ve installed the package by adding the following to my `~/.doom.d/packages.el`.

``` emacs-lisp
(package! parseclj
    :recipe (:host github :repo "clojure-emacs/parseclj"))

(package! parseedn
    :recipe (:host github :repo "clojure-emacs/parseedn"))
```

Any help will be appreciated.

**Comment by plexus** [comment]
That one is new to me. Googling for that error messages yields a few Spacemacs issues, but without any real explanation or resolution.

- are you able to get a stack trace (try `toggle-debug-on-error` first)?
- do you get the error on a clean config? (so without doom, only parseclj)

**Comment by tad-lispy** [comment]
Hi, thanks for getting back to me.

I tried it on clean Emacs (started with `emacs --no-init`) and it worked without any issues. Further more, after restarting Emacs with Doom - it works now! 

As I understand Doom uses different package management system, so maybe the problem was caused by some installation issue? After I installed the package using MELPA on clean Emacs it started to work.

For the record this is the code I evaluated in Scratch buffer of clean Emacs:

``` emacs-lisp
(require 'package)
(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)

(package-initialize)
(package-install 'parseedn)

(parseedn-read-str "42")
```
